packages/backend/Dockerfile

```diff
@@ -1,18 +1,30 @@
-# This dockerfile builds an image for the backend package.
-# It should be executed with the root of the repo as docker context.
-#
-# Before building this image, be sure to have run the following commands in the repo root:
+# This dockerfile builds an image for the BIH backend package.
+# It should be executed within the root of the repo as docker context.
 #
+# Before building this image be sure to have run the following commands in the repo root:
 # yarn install
 # yarn tsc
 # yarn build
-#
-# Once the commands have been run, you can build the image using `yarn build-image`
 
-FROM container-registry.dev8.bip.va.gov/bip/node:14
+# Once the commands have been run, you can build the image using `yarn build-image`.
```

Comment:
I believe this should be 'yarn docker-local'

Reply:
@ThomasQuinnKelly yarn docker-local will build the entire local docker-compose env. This command will build the backend image only, which is what I am looking for here. Thank you for comment though.